main.js

```diff
@@ -0,0 +1,15 @@
+// GABBY - PRINT TO DOM FUNCTION + LOOP FUNCTION 
+const printToDom = (divId, textToPrint) => {
+  const selectedDiv = document.querySelector(divId);
+  selectedDiv.innerHTML = textToPrint;
+}
+
+const createCards = (arr, card, id) => {
+  let domString = '';
+
+  for (let i=0; i < arr.length; i++) {
```

Comment:
I think this will need to be a for-each loop so that you can pass each item into the 'card' function.

Reply:
Would it work if each person has a variable called something like "overview-card" with the card inside, which we pass to the createCards function?

Reply:
Sorry, I meant a for-of loop. Something like this:
```for (item of arr) {
  domString += card(item);
}```